cypress/support/page-objects/ManualEventSyncPageObject.js

```diff
@@ -0,0 +1,48 @@
+import { dataTest } from "../utils";
+import ManualSyncPageObject from "./common/ManualSyncPageObject";
+
+class ManualEventSyncPageObject extends ManualSyncPageObject {
+    constructor(cy) {
+        super(cy, "events");
+    }
+
+    open() {
+        super.open("/#/sync/events");
+        return this;
+    }
+
+    displayOrgUnitChildren(orgUnit) {
+        this.cy
```

Comment:
The content of this method is enough generic, I think we could to create a new cypress command expandInOrgUnitTree similar to this: https://github.com/EyeSeeTea/metadata-synchronization/blob/e7ff2c0c546125db6cbc6eae65359659d4c38d20/cypress/support/commands.js#L89